include/libswiftnav/ephemeris.h

```diff
@@ -39,6 +42,34 @@ typedef struct {
 } ephemeris_xyz_t;
 
 typedef struct {
+  /*u8 m;*/     /* string number */
+  /*u8 B;*/     /* unhealthy SV indication, should be stored in ephemeris_t */
+  /*u8 n;*/     /* SV id , should be stored in ephemeris_t*/
+  /*u8 l;*/     /* unhealthy SV indication, should be stored in ephemeris_t */
+  /*u8 P;*/     /* SV operation mode in respect of time parameters TODO: decide is we need to store it */
+
+  u8 tk_hours;            /* Current frame start time since beginning of day */
+  u8 tk_minutes;          /* Not needed in ephemeris. TODO: decide if it needs to be stored here. */
+  u8 tk_half_minutes;
+
+  u16 tb;                 /* Index of the time interval within current day */
+  u8 sv_type;             /* Coded as word M in broadcasted ephemeris */
+  double gamma;
+  double tau;
+  double pos[3];          /* Position of the SV at tb in PZ-90.02 coordinates system, km */
+  double vel[3];          /* Velocity vector of the SV at tb in PZ-90.02 coordinates system, km/s */
+  double acc[3];          /* Acceleration vector of the SV at tb in PZ-90.02 coordinates system, km/s/s */
+  u16 NT;                 /* Current date, coded */
+  u16 accuracy;           /* User range accuracy coded in Ft word, meters */
+  double delta_tau;       /* Time difference between L1 and L2 signals */
+  u8 age;                 /* Age of the information. Time since calculation till tb in days. */
+  u8 P1;                  /* Size of a time interval for index tb, minutes */
+  u8 P2;                  /* Oddness ("1") or evenness ("0") of the value of tb for intervals 30 or 60 minutes */
+  u8 P3;                  /* A number of almanacs in given frame. "1" - for 5 SVs, "0" - for 4 SVs */
+  u8 P4;                  /* Flag indicating that a new ephemeris is present */
```

Comment:
We don't capture this for GPS, but we could. Will be useful along with `age` in the future. If we add, should put in the base struct